contracts/LinniaRecords.sol

```diff
@@ -72,31 +76,38 @@ contract LinniaRecords is Ownable {
     /// Add a record by user without any provider's signatures.
     /// @param fileHash the hash of the original unencrypted file
     /// @param recordType the type of the record
+    /// @param keywords keywords for the record
```

Comment:
consider improving the comment.  Maybe mentioning something like the following:
`this use used at different software layers for indexing`

Reply:
Also, can keywords be empty?